pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter.go

```diff
@@ -779,13 +779,42 @@ func validateSnapshotStatus(ar *admissionv1.AdmissionRequest, vm *v1.VirtualMach
 		}}
 	}
 
-	if !equality.Semantic.DeepEqual(oldVM.Spec, vm.Spec) {
+	if !compareDisks(oldVM.Spec.Template.Spec.Domain.Devices.Disks, vm.Spec.Template.Spec.Domain.Devices.Disks) ||
+		!compareVolumes(oldVM.Spec.Template.Spec.Volumes, vm.Spec.Template.Spec.Volumes) {
 		return []metav1.StatusCause{{
 			Type:    metav1.CauseTypeFieldValueNotSupported,
-			Message: fmt.Sprintf("Cannot update VM spec until snapshot %q completes", *vm.Status.SnapshotInProgress),
+			Message: fmt.Sprintf("Cannot update VM disks or volumes until snapshot %q completes", *vm.Status.SnapshotInProgress),
 			Field:   k8sfield.NewPath("spec").String(),
 		}}
 	}
 
 	return nil
 }
+
+func compareDisks(old, new []v1.Disk) bool {
```

Comment:
Not sure we care about disks only volumes: https://github.com/kubevirt/kubevirt/blob/b62599eee3960fb389330c870d1ca71a987e6a21/pkg/storage/snapshot/source.go#L359